### ISSUE TITLE
Create black box mode for research

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-basics#6]
Timeline was discussed at Dec 16 Meeting:

> March 1 …. update time. March 28 is beginning of the last term. Study could be between there and June 1. 

Kathy suggested: So perhaps sometime in May?

This would just be the black box, wouldn't need energy screen, etc.  Start with the playground screen with basic circuits (no energy components), the black box screen (for students), and the black box construct screen (for researchers to design black boxes).

**Comment by samreid** [comment]
Researchers should be able to create their own black boxes and choose the order they will appear in the combo box.

**Comment by samreid** [comment]
@kathy-phet said:
> electrons are not a priority for black box cck

**Comment by samreid** [comment]
> Kathy suggested: So perhaps sometime in May?

This will be difficult to get done in May unless I start on it really soon.  I'll mark this issue for phet-io meeting since phet-io is my main issue to clean up before starting on this.

**Comment by samreid** [comment]
Today @kathy-phet suggested taking 1-2 weeks to tidy up PhET-iO before switching to CCK.  That would be a start date no later than Feb 26.

**Comment by samreid** [comment]
I'm planning to start working on this on Friday.

**Comment by samreid** [comment]
This project is underway and the outstanding work is tracked in open issues, closing.
